xml/deploy_cephadm.xml

```diff
@@ -643,4 +643,95 @@ config: OK
    </tip>
   </sect2>
  </sect1>
+ <sect1 xml:id="deploy-cephadm-day2">
+  <title>Further Deployment and Administration (Day 2)</title>
+
+<!-- 2020-04-20 tbazant: seems it boils down to:
+   * listing services/devices
+   * host manipulation
+   * deploy services
+  -->
+
+  <para>
+   After you deployed the basic &ceph; cluster, you need to deploy core
+   services on more cluster nodes. To make the cluster data accessible to
+   clients, deploy additional services.
+  </para>
+
+  <tip>
+   <title>&cephsalt; and <command>ceph orch</command> CLI</title>
+   <para>
+    While &cephsalt; helps you configure basic aspects of the cluster as
+    outlined in <xref linkend="deploy-cephadm-day1"/>, you will need it further
+    on either to modify the basic configuration or add a new host to the
+    cluster.
+   </para>
+   <para>
+    The &ceph; orchestrator command <command>ceph orch</command>&mdash;which is
+    an interface to the &cephadm; module&mdash;will take care of listing
+    cluster components and deploying &ceph; services on new cluster nodes.
+   </para>
+  </tip>
+
+  <sect2 xml:id="deploy-cephadm-day2-orch">
+<!-- https://docs.ceph.com/docs/master/mgr/orchestrator/ -->
+   <title>The <command>ceph orch</command> Command</title>
+   <para></para>
+  </sect2>
```

Comment:
Is this section to be fleshed out?

Reply:
i'm workin gon it right now, i first wanted to include the upgrade stuff so that it can be online